x-pack/plugins/ml/common/constants/settings.ts

```diff
@@ -5,3 +5,5 @@
  */
 
 export const FILE_DATA_VISUALIZER_MAX_FILE_SIZE = 'ml:fileDataVisualizerMaxFileSize';
+export const ANOMALY_DETECTION_ENABLE_TIME_RANGE = 'ml:anomalyDetectionDefaultTimeRangeEnable';
```

Comment:
Setting the name this way because seems like the settings are ordered by these strings alphabetically.